extensions/reviewed/AutoTyping.json

```diff
@@ -7,22 +7,12 @@
   "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLXR5cGV3cml0ZXIiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMjAgMTNIMTZDMTYgMTQuMSAxNS4xIDE1IDE0IDE1SDEwQzguOSAxNSA4IDE0LjEgOCAxM0g0TDIgMThWMjBDMiAyMS4xIDIuOSAyMiA0IDIySDIwQzIxLjEgMjIgMjIgMjEuMSAyMiAyMFYxOE02IDIwQzUuMTEgMjAgNC42NiAxOC45MiA1LjI5IDE4LjI5QzUuOTIgMTcuNjYgNyAxOC4xMSA3IDE5QzcgMTkuNTUgNi41NSAyMCA2IDIwTTEwIDIwQzkuMTEgMjAgOC42NiAxOC45MiA5LjI5IDE4LjI5QzkuOTIgMTcuNjYgMTEgMTguMTEgMTEgMTlDMTEgMTkuNTUgMTAuNTUgMjAgMTAgMjBNMTQgMjBDMTMuMTEgMjAgMTIuNjYgMTguOTIgMTMuMjkgMTguMjlDMTMuOTIgMTcuNjYgMTUgMTguMTEgMTUgMTlDMTUgMTkuNTUgMTQuNTUgMjAgMTQgMjBNMTggMjBDMTcuMTEgMjAgMTYuNjYgMTguOTIgMTcuMjkgMTguMjlDMTcuOTIgMTcuNjYgMTkgMTguMTEgMTkgMTlDMTkgMTkuNTUgMTguNTUgMjAgMTggMjBNMTggMTBWM0g2VjEwSDNWMTJIMjFWMTBNOCA1SDE2VjZIOE04IDdIMTRWOEg4IiAvPjwvc3ZnPg==",
   "name": "AutoTyping",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/typewriter.svg",
-  "shortDescription": "Animate text to simulate it being written one character at at time (a \"typewriter\" effect).",
-  "version": "1.1.0",
+  "shortDescription": "Reveal a text one letter after the other.",
+  "version": "1.1.1",
   "description": [
-    "Animate text to simulate it being written one character at at time (a \"typewriter\" effect), with a customizable time between each character. Useful for dialogue scenes or visual novels.",
+    "Reveal a text one letter after the other (a \"typewriter\" effect).",
     "",
-    "How to use:",
-    "- Add the behavior to a Text object (BBText, Bitmap Text object) and choose the interval between characters.",
-    "",
-    "Tips:",
-    "- Change the typing speed with the \"Change interval\" action.",
-    "- Pause the typing and resume it at any time.",
-    "- Use the \"Skip to end of text\" action to give users a way to skip text they don't want to read ",
-    "- If the text changes, the automatic typing starts again from the beginning with the new text.",
-    "- Use a condition to check if the typing finished.",
-    "- Use a condition to check if a new text character was added. This is useful to trigger audio effects.",
-    "- Use the \"CurrentCharacter\" expression to know when a specific character was displayed.",
```

Comment:
This is a listing of the condition and expression. Their descriptions are already clear.